sql/lesson1/tutorial.md

```diff
@@ -0,0 +1,292 @@
+
+### People table
+
+people (id, name, age, country_of_birth, country_of_residence, number_of_children, studies, has_job)
+
+| id       | name                    | age   | country_of_birth | country_of_residence | number_of_children | studies         | has_job |
+| :------: | :---------------------: | :---: | :--------------: | :------------------: | :----------------: | :-----:         | :-----: |
+|1         | Emma Smith              |18     |UK                |UK                    |1                   |Computer science | yes     |
+|2         | Noah Johnson            |21     |UK                |US                    |0                   |Accountancy      | yes     |
+|3         | Luis Marco Polo         |23     |ES                |UK                    |0                   |Maths            | yes     |
+|4         | Olivia Juanes           |21     |AR                |PT                    |1                   |Arts             | yes     |
+|5         | William Brown           |25     |BR                |BR                    |1                   |Photography      | yes     |
+|6         | Sophia Davis            |30     |PT                |UK                    |2                   |NULL             | no      |
+|7         | Jacob Miller            |35     |NL                |NL                    |2                   |Graphic designer | yes     |
+|8         | Mason Wilson            |42     |FR                |FR                    |3                   |NULL             | yes     |
+|9         | Mia Moore               |50     |US                |UK                    |4                   |Law              | yes     |
+|10        | Abigail Moore           |28     |CA                |ES                    |2                   |Law              | yes     |
+|11        | Michael Anderson        |22     |IT                |IT                    |1                   |Human Resources  | no      |
+|...       | ...                     |...    |...               |...                   |...                 |...              | yes     |
+|40        | Benjamin Martinez Lopez |30     |ES                |IT                    |1                   |Computer science | yes     |
+
+## Write the following queries
+
+Open this fiddle that will have the data shown in the tables above already loaded: https://sqliteonline.com/#fiddle-5be8825d749e251ljod9s39t
+
+* L1.1 Show all the data available of all the people.
+* L1.2 Show the name and Country of residence of all the people 23 years old or above.
+* L1.3 Show the name and age of all the people living in UK.
+* L1.4 Show the name and age of all the people not living in their birth place.
+* L1.5 Show the distinct countries of residence of all the people.
+* L1.6 Show the distinct studies of the people having less than 2 children.
+
+We recommend you to try the above exercises in https://sqliteonline.com, if you get blocked, you can find the solutions [here](solutions.md)
```

Comment:
Recommend replacing `blocked` with `stuck`

Reply:
Replaced in the 6 lessons